playground/src/fly-in/fly-in-background.js

```diff
@@ -14,11 +14,18 @@
 
 import './fly-in-background.scss';
 import events from '../events/events.js';
+import modes from '../modes/index.js';
 
 export const EVENT_FLY_IN_CLOSE = 'event-fly-in-close';
 
+export function createFlyInBackground() {
```

Comment:
This is a bit weird as it now always will create something

Reply:
You probably reviewed a split-second too early, this is now also guarded by a mode-check.